sdk/core/azure-core/src/http/curl/curl_connection_pool_private.hpp

```diff
@@ -29,102 +29,104 @@ namespace Azure { namespace Core { namespace Test {
   class CurlConnectionPool_connectionPoolTest_Test;
   class CurlConnectionPool_uniquePort_Test;
   class CurlConnectionPool_connectionClose_Test;
+  class SdkWithLibcurl_globalCleanUp_Test;
 }}} // namespace Azure::Core::Test
 #endif
 
 namespace Azure { namespace Core { namespace Http { namespace _detail {
 
-  /**
-   * @brief CURL HTTP connection pool makes it possible to re-use one curl connection to perform
-   * more than one request. Use this component when connections are not re-used by default.
-   *
-   * This pool offers static methods and it is allocated statically. There can be only one
-   * connection pool per application.
-   */
-  class CurlConnectionPool final {
+    /**
+     * @brief CURL HTTP connection pool makes it possible to re-use one curl connection to perform
+     * more than one request. Use this component when connections are not re-used by default.
+     *
+     * This pool offers static methods and it is allocated statically. There can be only one
+     * connection pool per application.
+     */
+    class CurlConnectionPool final {
 #if defined(TESTING_BUILD)
-    // Give access to private to this tests class
-    friend class Azure::Core::Test::CurlConnectionPool_connectionPoolTest_Test;
-    friend class Azure::Core::Test::CurlConnectionPool_uniquePort_Test;
-    friend class Azure::Core::Test::CurlConnectionPool_connectionClose_Test;
+      // Give access to private to this tests class
```

Comment:
There seems to be unnecessary whitespace changes in this file. Revert those?

Reply:
done, fro fun i used the clang-format-11 tool in wsl to format the file, the previous one was VS autoformatting the file